aicoe/sesheta/chatbot.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# Sefkhet-Abwy
+# Copyright(C) 2019 Christoph Görn
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This is a Google Hangouts Chat Bot."""
+
+
+import os
+import asyncio
+import pathlib
+import logging
+
+
+from aiohttp import web
+
+
+from thoth.common import init_logging
+
+
+from aicoe.sesheta.utils import notify_channel, hangouts_userid, realname
+
+
+__version__ = "0.1.0-dev"
+
+
+init_logging()
```

Comment:
`THOTH_LOG_` will not work. A prefix argument needs to be passed.

Reply:
added by 819e80c7eef5015c4a81b4aa2aa67d0e53be30b5